torch/quantization/quantize.py

```diff
@@ -240,6 +258,11 @@ def convert(module, mapping=DEFAULT_MODULE_MAPPING):
     module_swapped = swap_module(module, mapping)
 
     reassign = {}
+    # TODO(jerryzh): remove after deciding on the impl of
+    # intrinsic moudles
+    if type(module) in [nni.ConvBn2d, nni.ConvBnReLU2d]:
```

Comment:
Why only these two modules? What about nni.ConvRelu2d and nni.LinearRelu2d?

Reply:
yeah we need these as well